### DISCUSSION TITLE
VitePress plugin for previewing templ components

### DISCUSSION THREAD

**Discussion by indaco** [a-h/templ#853] [Show and tell]
Hi all,
Just to inform you that I have released a VitePress plugin to preview your ` components while documenting them.

Check out the docs [here](https://vitepress-templ-preview.indaco.dev/). You will find examples and a getting started guide to help you. The docs website is built using this plugin.

The code is available [here](https://github.com/indaco/vitepress-templ-preview).

I would greatly appreciate it if someone could test it and share feedbacks, advices, and any issues you encounter.

Hope someone will find it useful.

Thank you!

**Comment by Riu** [comment]
Respect! Good job!
